jupiterone/resource_rule.go

```diff
@@ -228,9 +228,7 @@ func (*QuestionRuleResource) Schema(ctx context.Context, req resource.SchemaRequ
 						"actions": schema.ListAttribute{
 							Required:    true,
 							ElementType: types.StringType,
-							Validators: []validator.List{
```

Comment:
The when object above also has a jsonValidator for type string. Does it not have the same issue when using json encode?

Reply:
It did not, but I did a bit more testing and verified we don't need the custom validator at all.